From 0081d68c542df122e022db7ce3e77fc45302abe0 Mon Sep 17 00:00:00 2001
From: Pavani Reddy <pavani@broadcom.com>
Date: Mon, 24 Mar 2014 12:43:37 +0530
Subject: [PATCH 0496/1532] hybrid_nae_ucore : changed to poe distibution and
 port fifo mode instead of destination VC mode.

[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/mips/boot/dts/xlp9xx-hybrid-nae-helinux.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/mips/boot/dts/xlp9xx-hybrid-nae-helinux.dts b/arch/mips/boot/dts/xlp9xx-hybrid-nae-helinux.dts
index 841ecd7..bc4ecee 100644
--- a/arch/mips/boot/dts/xlp9xx-hybrid-nae-helinux.dts
+++ b/arch/mips/boot/dts/xlp9xx-hybrid-nae-helinux.dts
@@ -138,7 +138,7 @@
 				/* < node paddr-H32bits paddr-L32bits size-H32bits size-L32bits */
 				freein-fifo-replenish-addr-info = <0x0 0x0 0x88000000 0x0 0x10000000>; /* 256M @ 2176M*/
 				/* < node fifomask descsize headroom ndescs > */
-				freein-fifo-replenish-desc-info = <0 0x00ff 2048 0 64>;
+				freein-fifo-replenish-desc-info = <0 0x00ff 2048 0 150>;
 			};
 			nae-1 {
 				freein-fifo-mask = <0x00ff 0 0 0>;
@@ -153,7 +153,7 @@
 				/* < node paddr-H32bits paddr-L32bits size-H32bits size-L32bits */
 				freein-fifo-replenish-addr-info = <0x0 0x0 0x98000000 0x0 0x10000000>; /* 256M @ 2432M*/
 				/* < node fifomask descsize headroom ndescs > */
-				freein-fifo-replenish-desc-info = <0 0x00ff 2048 0 64>;
+				freein-fifo-replenish-desc-info = <0 0x00ff 2048 0 150>;
 			};
 
 			app-param{
@@ -215,7 +215,7 @@
 				/* < node paddr-H32bits paddr-L32bits size-H32bits size-L32bits */
 				freein-fifo-replenish-addr-info = <0x1 0x3 0x00c00000 0x0 0x10000000>; /* 256M @12300m*/
 				/* < node fifomask descsize headroom ndescs > */
-				freein-fifo-replenish-desc-info = <1 0xffff 2048 0 64>;
+				freein-fifo-replenish-desc-info = <1 0xffff 2048 0 150>;
 
 
 			};
-- 
1.9.1

